config/config.exs

```diff
@@ -6,6 +6,11 @@ config :braintree,
   public_key: System.get_env("BRAINTREE_PUBLIC_KEY"),
   private_key: System.get_env("BRAINTREE_PRIVATE_KEY")
 
+if Mix.env == :test do
```

Comment:
The config file is only used for testing, so this guard can be removed. Applications only load their own configuration, so this won't be loaded by any application that uses the package.